collector/__init__.py

```diff
@@ -251,6 +241,9 @@ def _instrument_setup(configuration: Dict[str, List[str]], instrument_name: str)
                 disperser = ulist[0]
             else:
                 instconfig[key] = ulist
+            
+            if 'Alopeke' in instrument_name:
+                instrument_name = 'Alopeke'
```

Comment:
Little confused by this. Is there any case where `Alopeke` would be IN the `instrument_name` but `instrument_name` would contain MORE than `Alopeke`?

Since my current codebase only runs for GS right now, I can't check it myself since Alopeke is only in GN.

Reply:
I figured that might be the case. I worked on the Alopeke implementation and I know that it can contain all kinds of additional information.